src/main/java/com/example/nlern/models/MarkDetail.java

```diff
@@ -3,6 +3,8 @@
 
 import java.util.List;
 
+import static com.example.nlern.Constants.NUMBER_OF_QUESTIONS;
```

Comment:
I strongly advise you not to use static imports unless readability suffers. We pollute imports by doing so.

Reply:
Done